### PR TITLE
Document how to embeds config inside a dependency

### PR DESCRIPTION
**Pull request by loicmathieu**

### CODE REVIEW

File: docs/src/main/asciidoc/config-reference.adoc [hunk 1]
```diff
@@ -86,6 +86,15 @@ quarkus.http.port=9090 <2>
 
 TIP: Quarkus supports the use of <<using_property_expressions,property expressions>> in the `application.properties` file.
 
+== Embedding a configuration file inside a dependency
+
+You can embed a configuration file inside one of your dependency by adding a `META-INF/microprofile.properties` configuration file to it
```
Comment (radcortez) [inline docs/src/main/asciidoc/config-reference.adoc:91]:
```suggestion
You can embed a configuration file inside one of your dependencies by adding a `META-INF/microprofile.properties` configuration file to it
```

**Review by radcortez — APPROVED** [review]

**Comment by canidae** [comment]
The documentation is incorrect. The file must be named `META-INF/microprofile-config.properties`, not `META-INF/microprofile.properties`. @loicmathieu

**Comment by loicmathieu** [comment]
@canidae right, I'll make the correction

**Comment by loicmathieu** [comment]
@canidae thanks for spotting the error :)

**Pull request merged**